types/params.go

```diff
@@ -132,6 +132,10 @@ func DefaultABCIParams() ABCIParams {
 }
 
 func IsValidPubkeyType(params ValidatorParams, pubkeyType string) bool {
+	// disallow more than one pubkey type
+	if len(params.PubKeyTypes) > 1 {
```

Comment:
let me know of this is fine

Reply:
why are we double-checking the length here again?